FCMExternalProberTarget/app/build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
I don't know much about Android standards.  So don't switch this until some research has been done.  Would Bazel be more appropriate for this?  We should confirm that this is a requirement for //third_party